docs/versioned_docs/version-2.50.0-LTS/data-sources/oracledb.md

```diff
@@ -5,49 +5,81 @@ title: Oracle DB
 
 # Oracle DB
 
-ToolJet can connect to Oracle databases to read and write data. 
+ToolJet can connect to Oracle databases to read and write data.
 
 ## Connection
 
-To establish a connection with the Oracle DB data source, click on the `+Add new` button located on the query panel or navigate to the [Data Sources](https://docs.tooljet.com/docs/data-sources/overview) page from the ToolJet dashboard.
+To establish a connection with the Oracle DB data source you can either click on the `+Add new` button in the query panel, or navigate to the [Data Sources](https://docs.tooljet.com/docs/data-sources/overview) page from the ToolJet dashboard.
+
+### Required Credentials
 
-A Oracle DB can be connected with the following credentials:
 - **Host**
 - **Port**
-- **SID / Service Name** ( Database name must be a SID / Service Name )
+- **SID / Service Name** (Database name must be a SID / Service Name)
 - **Database Name**
 - **SSL**
 - **Username**
 - **Password**
-- **Client Library Path** ( Only required for local setup )
+- **Client Library Path** (Only required for local setup)
```

Comment:
can remove `(Only required for local setup)`

For cloud deployments, you can add the client library to a custom directory or mount it as a volume in the container.
For local setups, specify the path to the Oracle client library here. This allows ToolJet to locate and use the necessary drivers for Oracle database connections.